### ISSUE TITLE
VirtualMachine power operations

### ISSUE THREAD

**Issue by bsherman** [snobear/ezmomi#32]
Implement basic VirtualMachine power operations:
- powerOn
- powerOff
- shutdown (use guest tools)

if shutdown is requested, but guest tools are not available, fall back to powerOff.

should wait for clean shutdown to complete for some amount of time ( 10 minutes ), then fall back to powerOff.

usage looks like:   
- ezmomi powerOn --name myVM
- ezmomi powerOff --name myVM
- ezmomi shutdown --name myVM

**Comment by bsherman** [comment]
I have an implementation for this, but it depends on currently pending, small PRs. So I'll wait on the PR till those are merged into develop.

**Comment by snobear** [comment]
implemented in #34
